src/Resources/translations/SonataMediaBundle.es.xliff

```diff
@@ -531,6 +531,14 @@
                 <source>form.label_class</source>
                 <target>form.label_class</target>
             </trans-unit>
+            <trans-unit id="form.label_icon">
+                <source>form_group.gallery</source>
+                <target>Galería</target>
+            </trans-unit>
+            <trans-unit id="form.label_icon">
```

Comment:
The id `form.label_icon` is used twice 🤔

Reply:
Fixed.